src/pages/document/[id].tsx

```diff
@@ -35,18 +34,21 @@ import { getMainDocuments } from "@helpers/getMainDocuments";
 
 import { sortFilterTargets } from "@utils/sortFilterTargets";
 import { pluralise } from "@utils/pluralise";
+import { getFamilyMetaDescription } from "@utils/getFamilyMetaDescription";
 
-import { TFamilyPage, TMatchedFamily, TTarget, TGeographySummary } from "@types";
+import { TFamilyPage, TMatchedFamily, TTarget, TGeography, TOrganisationDictionary } from "@types";
 
 import { QUERY_PARAMS } from "@constants/queryParams";
 import { EXAMPLE_SEARCHES } from "@constants/exampleSearches";
 import { MAX_FAMILY_SUMMARY_LENGTH } from "@constants/document";
 import { MAX_PASSAGES } from "@constants/paging";
+import { extractNestedData } from "@utils/extractNestedData";
 
 type TProps = {
   page: TFamilyPage;
   targets: TTarget[];
-  geographySummary: TGeographySummary;
+  countries: TGeography[];
```

Comment:
Do we want to stick with countries or use `geographies`?

Reply:
As its doing the same as the `useConfig` hook I've kept the naming consistent for now. I think the main reason we did that originally was to differentiate between the regions and countries

Reply:
Okie dokie :)